main/templates/main/about.html

```
@@ -23,65 +78,475 @@ <h5>Count details</h5>
+                </tr>
+            </tbody>
+        </table>
+
+        <p>Counts generated:</p>
+
+        <table class="table">
+            <thead>
+                <tr>
+                    <th>Count type</th>
+                    <th>Cancer grouping</th>
+                    <th class="text-end">c.807C>G</th>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)